<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Madde::Internal::MaemoPublishingUploadSettingsPageFremantleFree</class>
 <widget class="QWizardPage" name="Madde::Internal::MaemoPublishingUploadSettingsPageFremantleFree">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Upload Settings</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="accountNameLabel">
     <property name="text">
      <string>Garage account name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="garageAccountLineEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="getAccountLabel">
       <property name="text">
        <string>&lt;a href=&quot;https://garage.maemo.org/account/register.php&quot;&gt;Get an account&lt;/a&gt;</string>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;a href=&quot;https://garage.maemo.org/extras-assistant/index.php&quot;&gt;Request upload rights&lt;/a&gt;</string>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="keyFileLabel">
     <property name="text">
      <string>Private key file:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="Utils::PathChooser" name="privateKeyPathChooser" native="true"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="serverAddressLabel">
     <property name="text">
      <string>Server address:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="serverAddressLineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="targetDirectoryOnServerLabel">
     <property name="text">
      <string>Target directory on server:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="targetDirectoryOnServerLineEdit"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Utils::PathChooser</class>
   <extends>QWidget</extends>
   <header location="global">utils/pathchooser.h</header>
   <container>1</container>
   <slots>
    <signal>editingFinished()</signal>
    <signal>browsingFinished()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
